File: src/main/java/FlyAway/flight/dto/FlightDetailsClientDto.java
package FlyAway.flight.dto;

import FlyAway.flight.aircraft.dto.AircraftDto;
import FlyAway.flight.airport.dto.AirportBasicDto;

import java.time.LocalDateTime;

public record FlightDetailsClientDto(
        LocalDateTime departureDate,
        LocalDateTime arrivalDate,
        AirportBasicDto departureAirportDto,
        AirportBasicDto arrivalAirportDto,
        AircraftDto aircraftDto

) {
}

File: frontend/src/app/services/models/flight-details-client-dto.ts
/* tslint:disable */
/* eslint-disable */
import { AircraftDto } from '../models/aircraft-dto';
import { AirportBasicDto } from '../models/airport-basic-dto';
export interface FlightDetailsClientDto {
  aircraftDto?: AircraftDto;
  arrivalAirportDto?: AirportBasicDto;
  arrivalDate?: string;
  departureAirportDto?: AirportBasicDto;
  departureDate?: string;
}
